.gitattributes

```diff
@@ -0,0 +1,199 @@
+## GITATTRIBUTES FOR WEB PROJECTS
```

Comment:
Is this file needed for the tests? =)

Reply:
https://github.com/verdaccio/verdaccio/pull/1919#issue-474128576

Reply:
Ah, thanks. I forgot about this.

Is the merge button activated for you now?

Reply:
![github com_verdaccio_verdaccio_pull_1919](https://user-images.githubusercontent.com/6810177/91764696-0d7d4c00-ebd8-11ea-8092-da35a2edfb8b.png)

Reply:
Ok, then I'll merge this. Thanks for the feedback / screenshot.